Common/Models/WatchContext.swift

```diff
@@ -40,7 +40,6 @@ final class WatchContext: RawRepresentable {
     var recommendedBolusDose: Double?
 
     var potentialCarbEntry: NewCarbEntry?
-    var recommendedBolusDoseConsideringPotentialCarbEntry: Double?
```

Comment:
Removing this in favor of having a single "recommended bolus" passed in the context.